### ISSUE TITLE
universal viewer not loading?

### ISSUE THREAD

**Issue by malantonio** [LafayetteCollegeLibraries/spot#224]
![Screen Shot 2019-05-13 at 8 35 46 AM](https://user-images.githubusercontent.com/2744987/57621741-38164200-755a-11e9-84e1-95445a1f6c80.png)

for some reason the uv vendor directory isn't loading?

**Comment by malantonio** [comment]
looks like hyrax is in the process of changing how this is loaded since `pul_uv_rails` looks to be unmaintained. as a patch, i've been running:

```bash
cd $SPOT_BASE
cd public
git clone https://github.com/universalviewer/universalviewer
cd universalviewer
git checkout v2.0.1
```
though that will get removed when a new deploy is run.

**Comment by malantonio** [comment]
temporarily fixed with #228
